models/DataObject/ClassDefinition/Data/EncryptedField.php

```diff
@@ -591,21 +591,21 @@ public function isEqual($oldValue, $newValue): bool
 
     public function getParameterTypeDeclaration(): ?string
     {
-        return $this->delegate ? $this->delegate->getParameterTypeDeclaration() : null;
+        return null;
     }
 
     public function getReturnTypeDeclaration(): ?string
     {
-        return $this->delegate ? $this->delegate->getReturnTypeDeclaration() : null;
+        return null;
     }
 
     public function getPhpdocInputType(): ?string
     {
-        return $this->delegate ? $this->delegate->getPhpdocInputType() : null;
+        return $this->delegate ? $this->delegate->getPhpdocInputType() . "|\\Pimcore\\Model\\DataObject\\Data\\EncryptedField" : null;
```

Comment:
`sprintf('%s|\\%s', this->delegate->getPhpdocInputType() , EncryptedField::class)`

for better readability?

Reply:
Hi @dpfaffenbauer , changed this already because this was wrong anyway :-)